ui-src/src/assets/schema.json

```diff
@@ -41,18 +53,42 @@
     },
     "testSuite": {
       "type": "string",
-      "enum": ["Default-1", "Default-2", "Default-3"]
-    },
-    "requestDelay": { "type": "integer", "minimum": 1 },
-    "TPSFreq": { "type": "integer", "minimum": 1 },
-    "rampUsers": { "type": "integer", "minimum": 1 },
-    "rampDelay": { "type": "integer", "minimum": 0 },
-    "testCaseDir": { "type": "string" },
-    "testSuiteDir": { "type": "string" },
-    "baseStatsOutputDir": { "type": "string" },
-    "reportOutputDir": { "type": "string" }
+      "enum": [
+        "Default-1",
```

Comment:
these enums aren't valid, right? should there not be an array of test suite strings here?

Reply:
@ColmCooney is going to raise a defect for this